src/js/experimental/media-captions-listbox.js

```diff
@@ -39,16 +56,22 @@ class MediaCaptionsListbox extends MediaChromeListbox {
   attributeChangedCallback(attrName, oldValue, newValue) {
     super.attributeChangedCallback(attrName, oldValue, newValue);
 
-    if (attrName === MediaUIAttributes.MEDIA_SUBTITLES_LIST && oldValue !== newValue) {
+    if (
+      attrName === MediaUIAttributes.MEDIA_SUBTITLES_LIST &&
+      oldValue !== newValue
+    ) {
       this.#render();
-
-    } else if (attrName === MediaUIAttributes.MEDIA_SUBTITLES_SHOWING && oldValue !== newValue) {
+    } else if (
+      attrName === MediaUIAttributes.MEDIA_SUBTITLES_SHOWING &&
+      oldValue !== newValue
+    ) {
       this.value = newValue;
-
     } else if (attrName === 'aria-multiselectable') {
       // diallow aria-multiselectable
       this.removeAttribute('aria-multiselectable');
-      console.warn("Captions List doesn't currently support multiple selections. You can enable multiple items via the media.textTrack API.");
+      console.warn(
+        "Captions List doesn't currently support multiple selections. You can enable multiple items via the media.textTrack API."
```

Comment:
`media.textTracks` or just `TextTrack`

Reply:
Cool I didn't change that text but happy to update 😄 (change here is just another auto-reformatting)